packages/sdk-android/src/manifestParser.ts

```diff
@@ -114,7 +114,7 @@ const _mergeNodeParameters = (
         const key = k as NodeKey;
         const val = nodeParamsExt[key];
 
-        if (val && !SYSTEM_TAGS.includes(k)) {
+        if (val !== 'undefined' && !SYSTEM_TAGS.includes(k)) {
```

Comment:
@ElenaDiachenko can you clarify in description what does this line fixes. is it reproducible for @pauliusguzas @GabrieleKaceviciute ?